life50x50/Form1.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+
+namespace life50x50
+{
+    public partial class Form1 : Form
+    {
+        private bool[,] gameboard = new bool[50, 50];
+        private bool gamestarted = false;
+        private int stage;
+        private int changes;
+        private int totalAlive;
+        private Timer timer = new Timer();
+
+
+        Pen myPen = new Pen(Color.Gray);
+        Brush myBrush = new SolidBrush(Color.Black);
+        Graphics g = null;
+
+        public Form1()
+        {
+            InitializeComponent();
+            timer.Tick += new EventHandler(TimerEventProcessor);
+            timer.Interval = 1000;
+            menuStrip1.Items[1].Visible = false;
+            prepgame();
+        }
+
+        private void prepgame()
+        {
+            stage = 1;
+            totalAlive = 0;
+            changes = 0;
+            toolStripStatusLabel1.Text = "Поколение "+stage.ToString();
+            for (int i = 0; i < 50; i++)
+            {
+                for (int j = 0; j < 50; j++)
+                {
+                    gameboard[i, j] = false;
+                }
+            }
+            gamestarted = false;
+
+            panel1.Refresh();
+        }
+
+        private void выходToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            Close();
+        }
+
+        private void очиститьToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            prepgame();
+        }
+
+        private void panel1_Paint(object sender, PaintEventArgs e)
+        {
+            myPen.Width = 1;
+            g = panel1.CreateGraphics();
+            for (int i = 0; i<=1000; i = i+20)
+            {
+                g.DrawLine(myPen, i, 0, i, 1000);
+                g.DrawLine(myPen, 0, i, 1000, i);
+            }
+
+            for (int i = 0; i<50; i++)
+            {
+                for (int j = 0; j<50; j++)
+                {
+                    if (gameboard [i, j] == true)
+                    {
+                        g.FillEllipse(myBrush, i * 20, j * 20, 20, 20);
+                    }
+                }
+            }
+            
+            
+        }
+
+        private void panel1_MouseClick(object sender, MouseEventArgs e)
+        {
+            if (gamestarted == false)
+            {
+                if (gameboard[e.X / 20, e.Y / 20] == false)
+                {
+                    gameboard[e.X / 20, e.Y / 20] = true;
+                    totalAlive++;
+                    panel1.Refresh();
+                }
+            }

```

Comment:
Здесь уберем лишние скобки
```suggestion
            if (gamestarted == false)
                if (gameboard[e.X / 20, e.Y / 20] == false)
                {
                    gameboard[e.X / 20, e.Y / 20] = true;
                    totalAlive++;
                    panel1.Refresh();
                }

```